code/API_definitions/location-retrieval.yaml

```diff
@@ -87,7 +87,7 @@ info:
     # Further info and support
 
     (FAQs will be added in a later version of the documentation)
-  version: 0.3.0
+  version: wip

```

Comment:
also the server url should be set to wip: 
- url: '{apiRoot}/location-retrieval/vwip'
the same for location-verification